### ISSUE TITLE
Hyperactive ankle reflex

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5921]
**Preferred term label:** Hyperactive ankle reflex

**Synonyms** Hyperactive ankle jerk reflex, Hyperactive Achilles reflex

**Definition (free text, please give PubMed ID)** Hyperactive stretch reflex of the ankle.

**Parent term (use hpo.jax.org/app)** Lower limb hyperreflexia HP:0002395

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK396/

**Comment by pnrobinson** [comment]
added as HP:0033206
